### DISCUSSION TITLE
Welcome to ReactiveSearch Discussions!

### DISCUSSION THREAD

**Discussion by siddharthlatest** [appbaseio/reactivesearch#1566] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do.

**Comment by vmengh** [comment]
Hi @siddharthlatest 
This is a great library and works like a charm when there's elastic search backend serving the UI may be React/VueJS.

Have a question - 
> In case there's a need use the UI Search Component from appbase.io, can they be plugged in AS IS into any application that is NOT served from ElasticSearch but from some other data sources?

Appbase.io provides a feature library with faceted search & filters but appears to be tightly coupled with ElasticSearch

Thanks
